c8run/.env

```diff
@@ -4,5 +4,5 @@ CAMUNDA_VERSION=8.8.0-alpha2
 # Look here: https://artifacts.camunda.com/ui/native/connectors/io/camunda/connector/connector-runtime-bundle/
 CONNECTORS_VERSION=8.8.0-alpha2
 # version of docker compose artifact (used for --docker option)
-COMPOSE_TAG=8.7-alpha3
-COMPOSE_EXTRACTED_FOLDER=camunda-platform-8.7-alpha3
+COMPOSE_TAG=alpha-8.8
```

Comment:
The [PR for the compose 8.8](https://github.com/camunda/camunda-distributions/pull/104) is not yet merged/ released, right?

Reply:
I'm adding a commit to revert this part of the change / reference 8.7